plugins/@grouparoo/hubspot/src/lib/export/exportProfile.ts

```diff
@@ -1,10 +1,45 @@
-import { ExportProfilePluginMethod } from "@grouparoo/core";
+import {
+  ExportProfilePluginMethod,
+  Errors,
+  SimpleAppOptions,
+  DestinationSyncOperations,
+} from "@grouparoo/core";
+import { HubspotClient } from "../client";
 import { connect } from "../connect";
 import { addToList, removeFromList } from "./listMethods";
 
+const deleteContactOrClearGroups = async (
```

Comment:
I like the clarity here - we are going to either delete the contact or clear the groups.  'nothing' is not an option for any syncMode